### ISSUE TITLE
Bigger maps!

### ISSUE THREAD

**Issue by JoeMitchell** [OpenACalendar/OpenACalendar-Web-Core#318]
Everyone loves maps. So could they be a bit bigger? Twice as nice?

**Comment by jarofgreen** [comment]
> Everyone loves maps

Not sure that's true actually :-)

> So could they be a bit bigger? Twice as nice?

I did have them bigger before, but it just didn't look like the right balance with the other information there.

Does the "view bigger map" link underneath the maps not interest you? Does that need to be better visually linked to map maybe?

How about maps starting small and getting taller if they are interacted with? (If that is possible)

Interesting; does the map link undereath

**Comment by JoeMitchell** [comment]
Must be true!

The button underneath takes me to a new page, which I think users never
want - I want to stay where I am, but with a bigger map! Expanding maps
would be clever...(if that's possible - i looked at some of the examples on
leafletjs but couldn't spot anyone using it like that.

**Comment by jarofgreen** [comment]
Ok, so: Your looking at an event, like http://meetyournextmp.com/event/3-artists-union-england-hustings There's a map on the page. There is also another event, a week later, 500m down the road.

Q: While looking at the map on that page, would you expect to see a marker for the other event to?

**Comment by JoeMitchell** [comment]
Yeah, I think that's worth a try - in a colour that is duller / grey compared with the 'live' marker.

Just looking at that page, I half expect to see the map in the top right (it would nudge 'More details' down). In line with the sentence "Artists' Union England is very pleased to announce we are hosting a Hustings on the..."

The map could be repeated further down (i.e. you wouldn't have to then get rid of that map). What do you think?

**Comment by JoeMitchell** [comment]
Just to continue the map conversation...

I've added an event, but the pin is in the wrong place. I know I can edit the location detail later, but would be good to do it while adding the event...

It'd also be great to add some notes on the venue...again without having to first save the event, then go back and edit the location details?

**Comment by jarofgreen** [comment]
Adding notes while creating will go live soon.

Made separate issue for pin.

For Maps, pretty sure I don't want to have 2 maps. Interesting to think about moving up tho. At moment event links are privileged with top right space and I think that's more important. Would be good to sit with pen and paper and express ideas on the layout of this whole page in general later?

**Comment by JoeMitchell** [comment]
Yes, that's a good plan. Maybe on the Weds eve that you're down for?
